src/app/vault/vault.component.html

```diff
@@ -28,6 +28,9 @@ <h1>
                     </button>
                 </div>
             </div>
+            <div class="alert alert-warning" *ngIf="deleted" >
```

Comment:
use the `app-callout` component to keep styling in line with other places around the web vault